README.md

```diff
@@ -1,11 +1,96 @@
 # Capsule
 
-A framework for network function development. Written in Rust, inspired by [NetBricks](http://netbricks.io/) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
+A framework for network function development. Written in Rust, inspired by [NetBricks](https://www.usenix.org/system/files/conference/osdi16/osdi16-panda.pdf) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
+
+## Table of Contents
+
+* [Overview](#overview)
+* [Quick Start](#quick-start)
+* [Contributing](#contributing)
+* [Code of Conduct](#code-of-conduct)
+* [Contact](#contact)
+* [Maintainers](#maintainers)
+* [License](#license)
 
 ## Overview
 
-The project is currently undergoing heavy development. The goal is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for engineers. We are building a tool to efficiently manipulate network packets while being type-safe, memory-safe, and thread-safe.
+The goal of `Capsule` is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for developers. We've built a tool to efficiently manipulate network packets while being type-safe, memory-safe, and thread-safe. By combining `DPDK` with `Rust` Programming Language, `Capsule` offers
+
+* a fast packet processor that uses minimum number of CPU cycles.
```

Comment:
minor. I mad break it up like this:

* a fast packet processor that uses minimum number of CPU cycles;
* a rich packet type system that guarantees memory-safety and thread-safety;
* a declarative programming model that emphasizes simplicity; and,
* an extensible and testable framework that is easy to develop and maintain.

but that's just me.

Reply:
as discussed, we will keep it as is.